group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -2579,7 +2579,7 @@ public void replay(
         ConsumerGroup consumerGroup = getOrMaybeCreatePersistedConsumerGroup(groupId, false);
 
         if (value != null) {
-            consumerGroup.updateTargetAssignment(memberId, Assignment.fromRecord(value));
+            consumerGroup.updateTargetAssignment(memberId, Utils.fromRecord(value));
```

Comment:
Why are we doing this change? It seems better to keep `fromRecord` where it was.

Reply:
The method signature is more conversion from `ConsumerGroupTargetAssignmentMemberValue` to `Assignment` which seemed more aligned to me for Utils, hence made the change. If you think otherwise then I can move it back, I don't have strong opinion on this.

```
public static Assignment fromRecord(
        ConsumerGroupTargetAssignmentMemberValue record
    ) {

Reply:
Done.